tox.ini

```diff
@@ -8,7 +8,7 @@ envlist = py36
 
 [testenv]
 commands =
-    python -m unittest discover -s tests -p test_*.py
+    python -m {posargs:unittest discover -s tests -p test_*.py}
```

Comment:
:+1: Easier than  having to modify the tox file to try things, run a specific test only.

Reply:
Perhaps this should go into commontooling?

Reply:
I think that would be useful, yes.